docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+Rust
+====
+
+We've started the process of rewriting some of the
+performance-critical EdgeDB internals to Rust. So far we have
+converted the lexer as well as the GraphQL parser.
+
+Additionally, we're also making EdgeDB command-line tools using Rust.
+Notably the REPL has been converted and now features more commands:
+
+.. code-block::
+
+    Introspection
+      (options: -v = verbose, -s = show system objects,
+      -I = case-sensitive match)
+
+      \d [-v] NAME             describe schema object
+      \l, \list-databases      list databases
+      \lT [-sI] [PATTERN]      list scalar types
+      \lt [-sI] [PATTERN]      list object types
+      \lr [-I]                 list roles
+      \lm [-I]                 list modules
+      \la [-Isv] [PATTERN]     list expression aliases
+      \lc [-I] [PATTERN]       list casts
+      \li [-Isv] [PATTERN]     list indexes
+      \list-ports              list ports
+
+    Operations
+      \dump FILENAME           dump current database into a file
+      \restore FILENAME        restore the database from file into
+                               the current one
+      \create-database DBNAME  create a new database
+
+    Editing
+      \s, \history             show history
+      \e, \edit [N]            spawn $EDITOR to edit history entry N
+                               then use the output as the input
+
+    Settings
+      \set [OPTION [VALUE]]    show/change setting
+
+    Connection
+      \c, \connect [DBNAME]    Connect to database DBNAME
+
+    Development
+      \E                       show most recent error message at
+                               maximum verbosity
+      \pgaddr                  show the network addr of the postgres
+                               server
+      \psql                    open psql to the current postgres
```

Comment:
This is also old output, we dropped this command from help text.

Reply:
Odd, I used my REPL help to get this (which should be pretty fresh). I'll pull the latest version.